sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -297,24 +352,27 @@ class HttpResponseError(AzureError):
     :vartype error: ODataV4Format
     """
 
-    def __init__(self, message=None, response=None, **kwargs):
+    def __init__(
+        self, message: Optional[object] = None, response: "Optional[_HttpResponseCommonAPI]" = None, **kwargs: Any
```

Comment:
does _HttpResponseCommonAPI need to be a forward ref here and below?

Reply:
Good point